barretenberg/cpp/src/barretenberg/commitment_schemes/utils/mock_witness_generator.hpp

```diff
@@ -85,6 +89,10 @@ template <typename Curve> struct MockWitnessGenerator {
 
         polynomial_batcher.set_unshifted(RefVector(unshifted_polynomials));
         polynomial_batcher.set_to_be_shifted_by_one(RefVector(to_be_shifted_polynomials));
+
+        claim_batcher =
```

Comment:
You'll notice that I used the arguably simpler designated initializer pattern for the ClaimBatcher. I opted not to do this for PolynomialBatcher because it requires a class be very simple (all public members, no explicitly defined constructors). The PolynomialBatcher requires more complicated initialization that sets metadata (`full_batched_size`) and initializes the batched polynomials, and this can only be done through an explicit constructor. Still, the lack of symmetry between these two somewhat analogous classes is annoying and I may try to find a way to make them resemble each other a bit more.